source/includes/config_repos/_00-describe.md.erb

```diff
@@ -0,0 +1,8 @@
+<%=
+describe_object 'The config repo object', since: '17.8.0' do
```

Comment:
we are announcing it as a part of 17.9
[this](https://github.com/gocd/api.go.cd/pull/185/commits/5cc1faaa4ce84e96a9e1ff59a8a1025943e86504) comment has changed the version from `17.8.0` to `17.9.0`